### DISCUSSION TITLE
Window should be draggable by the strip on the left

### DISCUSSION THREAD

**Discussion by dwt** [wireapp/wire-desktop#5065] [Ideas]
Wire version: Version 3.1.2822
Wire for web version: Wire für Web Version 2018.02.16.0754
Operating system: Mac OS 10.13.3 (17D102)

What steps will reproduce the problem?
1. Look at the wire window, which has this huge strip on the left that looks as if it is the window header
2. Try to grab the window and drag it by that strip as there is pretty much nothing else evailable (visually)
3. Fail and be annoyed.

What is the expected result?
That strip looks like the window header, is the only (visually) clean part of the window, and therefore the window should be draggable by it.

What is the actual result?
It's not.

![bildschirmfoto 2018-03-07 um 09 51 33](https://user-images.githubusercontent.com/57199/37082637-58de1ae2-21ed-11e8-8b85-6008932d6bb0.png)

**Comment by gregor** [comment]
Thanks for the feedback. In the nature of this being a feature request I am going forward with closing the issue at this time.
